example_workflows/python_population_analysis/generate_csv.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+from csv import DictWriter
+
+from faker import Faker
```

Comment:
Saw that CVDNet were looking at [this](https://mimesis.name/master/about.html) for fake data. Not really important for a toy example though.